app/presenters/service_manual_guide_presenter.rb

```diff
@@ -8,7 +8,7 @@ def body
 
   def header_links
     header_links = details.fetch("header_links", {})
-    Array(header_links).map { |h| ActiveSupport::HashWithIndifferentAccess.new(h) }
+    Array(header_links).map { |h| ActiveSupport::HashWithIndifferentAccess.new(h.transform_keys { |k| k == "title" ? "text" : k }) }
```

Comment:
I don't think this is the best way of doing this, very open to suggestions that make it clearer, or I can add some comments